docs/override_check.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/lib/env python
+#
+# Module Documentation Override Check
+#
+# Verifies that any modules present in the _templates/overrides directory have all of their
+# their exported functions included in the doc file.
+
+from __future__ import print_function
+
+import glob
+import importlib
+import os
+import sys
+
+
+failed = False
+for full_path in glob.glob('_templates/overrides/metpy.*.rst'):
+
+    filename = os.path.basename(full_path)
+    module = filename.split('.rst')[0]
+
+    # Get all functions in the module
+    i = importlib.import_module(module)
+    functions = set(i.__all__)
+
+    # Get all lines in the file
+    with open(full_path) as f:
+        lines = f.read().splitlines()
+    lines = {line.strip() for line in lines}
+
+    # Check for any missing functions
+    missing_functions = functions - lines
+
+    if len(missing_functions) > 0:
```

Comment:
This can just be `if missing_functions:`, but I don't want to hold this up for that.